driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -42,12 +42,7 @@
 import java.io.PrintWriter;
 import java.io.StringWriter;
 import java.net.InetSocketAddress;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.LinkedHashSet;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Not a good idea. It's better to be explicit.

Reply:
This is something that maven automated formatter did.
Should I be calling something different than `mvn com.coveo:fmt-maven-plugin:format` suggested by the tool?

Reply:
If that's how the formatter is configured then let's leave it as is.

Reply:
I've readded explicit imports and this time it was satisfied with reordering. I'm not sure anymore what caused that but I'll keep an eye on such changes in the future.